tools/mason/MasonPublish.chpl

```diff
@@ -78,6 +79,35 @@ proc masonPublish(ref args: list(string)) throws {
       args.append(potentialPath);
     }
 
+    if createReg {
+      var pathReg = '';
+      if args[args.size-1] == '-c' || args[args.size-1] == '--create-registry' then
```

Comment:
This seems fragile (e.g. if this subcommand supports a new another arg someday, this code must be updated). Why don't we just find which arg has `-c` or `--create-registry` and then assign `pathReg` to that index + 1?

Reply:
@ben-albrecht  - But `pathReg` can come before/after the option `-c` right ? How would we decide `idx+1` or `idx-1` for `pathReg` ?

Reply:
> But `pathReg` can come before/after the option `-c` right ? 

Good point. We could search for any positional arguments that don't start with `-` in `args[2..]` for the path and use that index. If multiple positional args are found, that's an error.

Reply:
> If multiple positional args are found, that's an error.

But doesn't this contradict with :
 >> supports a new another arg someday,

Reply:
Yes 😄 . I was thinking more of flags though, e.g. `mason publish --force --create-registry foo`.

Supporting a new positional argument would be a much bigger change, unless it was simply supporting multiple registries, which wouldn't be so hard to get to from the above suggestion.